nx-X11/programs/Xserver/hw/nxagent/man/nxagent.1

```diff
@@ -441,9 +441,22 @@ enable proxy binding mode
 .B \-options \fIfilename\fP
 path to an options file containing nx/nx options (see below).
 .PP
-Other than the command line options, \fBnxagent\fR can be configured at
-session startup and at runtime (i.e. when resuming a suspended session)
-by so-called nx/nx options.
+Other than the command line options, \fBnxagent\fR can be configured
+at session startup and at runtime (i.e. when resuming a suspended
+session) by so-called nx/nx options. These options are always passed
+as a (sometime looong) options string consisting of option=value
+pairs, separated by commas. \fBnxagent\fR will extract that
+options string from the DISPLAY variable (see below) or an options
+file (provided via the \-\fIoptions\fR command line option). To
```

Comment:
Or an **options string** passed on the command line?

Reply:
I have left that out on purpose:
1. the text is already confusing, I did not want to add more confusion.
2. the manpage does not include any hint that this is possible at all (which should probably be changed; I'll open an issue for that)
3. an options string passed via cmdline is mainly usable for non-persistent sessions and testing. So it is nothing for the majority of users

Further, I'am wondering if we should drop even "options via DISPLAY" out of the equation, see #564.

Reply:
Yes and no... the rules for this stuff are quite complicated and currently not documented - mainly the overrides.

I guess we can leave that out of this PR and only mention the most commonly used scheme, yes.